### DISCUSSION TITLE
Unable to locate start.sh

### DISCUSSION THREAD

**Discussion by Gulraiz832** [emqx/emqx#7664] [Q&A] [answered]
Hi, I have updated my emqx from 3.2.3 to 4.4.3. After upgrading I have noticed that there is no start.sh in "/usr/bin". I was using it to add and remove users. Does anyone know its new location?

**Comment by zmstone** [comment]
Hi @Gulraiz832 how did you upgrade it?
How is the 4.4.3 package installed ?

**Reply by Gulraiz832** [reply]
@zmstone  I am using a docker image. So I have just changed the tag. Previously I was copying my start.sh to /usr/bin which overrode the existing start.sh

**Reply by zmstone** [reply]
If you are building your own image, please copy it to /opt/emqx/bin instead, maybe add a symlink from /usr/bin
And a warning: the boot script has been changed a lot since 4.2

**Comment by zmstone** [comment, marked as answer]
From 4.3 the bin path is /opt/emqx/bin
